src/microbe_stage/BiomeConditions.cs

```diff
@@ -8,6 +8,8 @@
 public class BiomeConditions : ICloneable, ISaveLoadable
 {
     public float AverageTemperature;
+    public float SpawnChunkiness;
+    public float SpawnRateMultiplier;
```

Comment:
You should put the default values here, so the json could leave out values (if wanted in the future) where they are the defaults.